src/stock-on-hand-mobile/pages/program-select.jsx

```diff
@@ -0,0 +1,116 @@
+import { RadioButton } from '../components/radio-button';
+
+import Select from '../../react-components/inputs/select';
+
+
+const ProgramSelect = ({ offlineService }) => {
+
+    const facility = useSelector(state => state[`facilitiesStockOnHand`][`userHomeFacilityStockOnHand`]);
+    
+    const programs = facility.supportedPrograms.map(({ id, name }) => ({ value: id, name }));
+
+    const menu = document.getElementsByClassName("header ng-scope")[0];
+    
+    const [facilityId, setFacilityId] = useState(null);
+    const [programId, setProgramId] = useState(null);
+    const [programName, setProgramName] = useState(null);
+    const [facilityType, setFacilityType] = useState("MyFacility");
+
+    const radioChangeHandler = (e) => {
+        setFacilityType(e.target.value);
+    };
+
+    useEffect(() => menu.style.display = "", [menu]);
+
+    return (
+        <div>
+            <div className="page-header-responsive">
+                <h2 id='program-select-header'>Stock on Hand</h2>
+            </div>
+            <div className="page-content">
+                <label 
+                  id='facility-type-header' 
+                  style={{marginBottom: "4px", fontFamily: "Arial", fontSize: "16px"}}
+                >
+                  Facility Type
+                </label>
+                <div>
+                    <RadioButton
+                        changed={radioChangeHandler}
+                        id="1"
+                        isSelected={facilityType === "MyFacility"}
+                        label={`My Facility `}
+                        additionalInfo={`(${facility.name})`}
+                        value="MyFacility"
+                    />
+                    <RadioButton
+                        changed={radioChangeHandler}
+                        id="2"
+                        isSelected={facilityType === "SupervisedFacility"}
+                        label="Supervised Facility"
+                        value="SupervisedFacility"
+                    />
+                </div>
+                <div style={{marginTop: "8px", marginBottom: "8px"}}>
+                    <div className='required' style={{marginBottom: "4px", fontFamily: "Arial", fontSize: "16px"}}>
+                      <label 
+                        id='facility-type-header'
+                      >
+                        Program
+                      </label>
+                    </div>
+                    <div className='field-full-width' style={{marginBottom: "8px"}}>
+                        <Select
+                          options={programs}
+                          onChange={value => setProgramId(value)}
+                        />
+                    </div>
+                    {facilityType === "SupervisedFacility" && (
+                      <>
+                        <div className='required' style={{marginBottom: "4px", fontFamily: "Arial", fontSize: "16px"}}> 
+                          <label 
```

Comment:
The same as above, label and id can be on the same line as there is no need to have separate line for props as there is only one